src/mixxxmainwindow.cpp

```diff
@@ -62,6 +62,8 @@
 #include "vinylcontrol/vinylcontrolmanager.h"
 #endif
 
+#include "cxx-qt-gen/rekordbox.cxxqt.h"
```

Comment:
This import in this file doesn't compile (I get no file or directory with this name error) whereas [importing in `main.cpp`](https://github.com/mixxxdj/mixxx/pull/12873/files#diff-34d21af3c614ea3cee120df276c9c4ae95053830d7f1d3deaf009a4625409ad2R25) correctly compiles and execute. Any ideas why?